### PR TITLE
E2E/MDW Governance - De-Identify dataset before move to Bronze w. Presidio

### PR DESCRIPTION
**Pull request by balteravishay**
# Type of PR

- Documentation changes
- Code changes
- CI-CD changes

## Purpose
This PR adds a stage to the modern-data-warehouse governance sample which anonymizes the description column of the dataset using Presidio. This stage is implemented using Azure Databricks, before the dataset is saved to Bronze container.

## Author pre-publish checklist

- [ ] Added test to prove my fix is effective or new feature works
- [ ] No PII in logs
- [v] Made corresponding changes to the documentation

### CODE REVIEW

File: e2e_samples/mdw_governance/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Modern Data Warehouse Data Governance end-to-end deployment with Purview <!-- omit in toc -->
 
-This sample demonstrates how to provision end-to-end modern data warehouse solution using [Azure DevOps pipelines](https://azure.microsoft.com/en-au/services/devops/pipelines/) to deploy across Dev, Test and Prod environments. It particularly focuses on data governance, using [Azure Purview](https://azure.microsoft.com/en-au/services/purview/), to track data assets and lineage to allow data managers to understand the quality and source of the data, as well as to understand if there is PII and other sensitive information in your data estate.
+This sample demonstrates how to provision end-to-end modern data warehouse solution using Azure DevOps pipelines to deploy Dev, Test and Prod environments.
+
+The sample focuses on data governance, using Purview, to track data assets and lineage to allow data managers to understand the quality and source of the data, as well as to understand if there is PII etc. in the data estate. It also uses [presidio](https://github.com/microsoft/presidio), to anonymize parts of the scanned data and redcue the potential to expose PII.
```
Comment (devlace) [inline e2e_samples/mdw_governance/README.md:5]: This seems to completely replace the above paragraph. Can you append the last sentence "It also uses presidio..." to the end of the paragraph instead? Also typo: reduce*

File: e2e_samples/mdw_governance/README.md [hunk 2]
```diff
@@ -20,9 +22,9 @@ This sample demonstrates how to provision end-to-end modern data warehouse solut
...
+This solution sets up an Azure Data Lake storage account, with two containers: Datalake and Dropzone. The folder structure in datalake is structured to enable data tiering (Bronze, Silver, Gold), hold shared data (Reference) and shared libraries (Sys). Azure Data Factory instance with linked services connecting to the Azure Data Lake, Azure Key Vault, Azure Databricks and Azure Purview. Application Insights is used for event logging and Office365 API Connection in combination with Logic Apps is used for sending notification emails. A Virtual Network and Private Endpoints are deployed for Data Lake and Key Vault, however the firewall on the services is left open in the initial deployment. Additionally the Databricks virtual network is not peered to the solution's and the databricks public IP is enabled. To fully secure the solution, the Virtual Network should be peered to an internal corporate network, and firewall closed on those services.
```
Comment (devlace) [inline e2e_samples/mdw_governance/README.md:25]: Could you also add a link to the existing single tech sample which demonstrates locking down a Databricks environment: https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/master/single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment

**Review by devlace — CHANGES_REQUESTED** [review]
Thanks @balteravishay! I've quickly reviewed the docs and left some comments. I'm still testing out the sample and will leave comments after then. Hoping to finish by early next week. Also, some of the PR checks (linting/link checks) are failing.

**Comment by balteravishay** [comment]
@devlace PR comments are resolved.

**Review by devlace — APPROVED** [review]

**Pull request merged**